### PR TITLE
FLOE-494: Upgraded to Infusion 2.0

### PR DESCRIPTION
**Pull request by jobara**
- Upgraded to Infusion 2.0
- Fixed a styling issue that prevented UIO from being visible with the Twenty Seventeen theme
- Restructured files by adding a src directory

https://issues.fluidproject.org/browse/FLOE-494

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -39,7 +39,13 @@ for tips on styling your site to work nicely with UIO.
 = Are there any themes themes that already work well with UIO? =
```
Comment (cindyli) [inline readme.txt]: double wording on `themes`. Please remove one. Thanks.

File: readme.txt [hunk 2]
```diff
@@ -39,7 +39,13 @@ for tips on styling your site to work nicely with UIO.
...
 
 The plugin has been tested with several themes, and we found that the Miranda theme responds very nicely.
+The default Twenty Seventeen theme also works fairly well, although some layout tweaks may be required for some preference combinations.
 
 == Issue Tracker ==
 
-http://issues.fluidproject.org/browse/FLOE/component/11510#selectedTab=com.atlassian.jira.plugin.system.project%3Acomponent-issues-panel
\ No newline at end of file
+http://issues.fluidproject.org/browse/FLOE/component/11510#selectedTab=com.atlassian.jira.plugin.system.project%3Acomponent-issues-panel
```
Comment (cindyli) [inline readme.txt]: This link doesn't work. Also, the link at line 36 points at an UIO doc on the wiki.fluidproject.org site. Shall it be updated to point to the infusion doc site?

File: src/php/inject-dependencies.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+Copyright 2015 OCAD University
```
Comment (cindyli) [inline src/php/inject-dependencies.php]: Fix the copyright. The same comment applies to other files.

Comment (jobara) [reply src/php/inject-dependencies.php]: I didn't really change the contents of most of the files, so I left the copyrights. For the most part I just moved them.

File: readme.txt (hunk 2 above)
Comment (jobara) [reply readme.txt]: The one at line 36 is only a placeholder in the Infusion Docs site, but this will future proof the readme a bit, so I can change it.

**Comment by jobara** [comment]
@cindyli updated and ready for more review.

**Comment by jobara** [comment]
@cindyli ready for more review and updated the [dev guide](https://wiki.fluidproject.org/display/fluid/UI+Options+Wordpres+Plugin+Development+Guide)

**Comment by cindyli** [comment]
Merged at 2967766f6896de5169e1e5a2b9c4ac08cde6563a

**Pull request merged**